.github/workflows/main.yml

```diff
@@ -0,0 +1,51 @@
+name: build
+on:
+ push:
+ pull_request:
+ schedule:
+#Every 50 days at midnight 
+    - cron:  "0 0 1/600 * *"
+
+jobs:
+  compilejobFedora:
+    name: APFEL_on_Fedora
+    runs-on: ubuntu-latest
+    container:
+        image: fedora:latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 3
+    - name: Install 
+      run: |
+           uname -a 
+           cat /etc/issue
```

Comment:
I guess these are debug prints. But if you wish to keep them for the sake of reporting, it's perfectly fine.

Reply:
removed.